.github/workflows/demo-doc.yml

```diff
@@ -1,34 +0,0 @@
-name: demo documentation build
```

Comment:
Why removing this, how are we going to build demo's doc site?

Reply:
I added a build of the demos into the standard docs build and it can be viewed at http://docs.djl.ai/docs/demos/index.html. This lets the single search on the docs site search the demos as well. As part of this, we should really remove the old demodocs.djl.ai site completely (maybe link it to the new site).